tests/js/spec/views/organizationEventsV2/index.spec.jsx

```diff
@@ -81,4 +91,65 @@ describe('OrganizationEventsV2', function() {
     const modal = wrapper.find('EventDetails');
     expect(modal).toHaveLength(1);
   });
+
+  it('navigates when tag values are clicked', async function() {
+    const organization = TestStubs.Organization({projects: [TestStubs.Project()]});
```

Comment:
I've been testing out this helper: `app-test/helpers/initializeOrg' (it could use a better name)

```suggestion
    const {organization, router, routerContext} = initializeOrg();
```